models/mealtopping.js

```diff
@@ -1,7 +1,7 @@
-'use strict';
 module.exports = (sequelize, DataTypes) => {
   const mealTopping = sequelize.define('mealTopping', {
-    id: DataTypes.INTEGER
+    mealCompositionId: { type: DataTypes.INTEGER, primaryKey: true },
+    toppingId: { type: DataTypes.INTEGER, primaryKey: true }
   }, {});
   mealTopping.associate = function(models) {
     // associations can be defined here
```

Comment:
you didn't have to associate this one with a `.belongsTo` to topping and mealComposition? like you did do on `mealComposition`?